state/enableha.go

```diff
@@ -8,6 +8,7 @@ import (
 	"strconv"
 
 	"github.com/juju/errors"
+	"github.com/juju/juju/environs/bootstrap"
```

Comment:
Should the juju/juju imports be in a separate group at the bottom? (For consistency with other files)

Reply:
yup, fixed